src/Network/Network/BYOIP/CustomIpPrefix/UpdateAzureCustomIpPrefixCommand.cs

```diff
@@ -67,7 +67,13 @@ public class UpdateAzureCustomIpPrefixCommand : CustomIpPrefixBaseCmdlet
         public SwitchParameter Commission { get; set; }
 
         [Parameter(Mandatory = false, HelpMessage = "Run cmdlet in the background")]
-        public SwitchParameter Decomission { get; set; }
+        public SwitchParameter Decommission { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "Run cmdlet in the background")]
+        public SwitchParameter Provision { get; set; }
```

Comment:
- I recommend using bool as the type of Provision so that we needn't two parameters for one function.
- Please update the help message.